src/languages.ts

```diff
@@ -35,7 +35,11 @@ export function parseLanguage(language: string): Language | undefined {
 }
 
 export function isTracedLanguage(language: Language): boolean {
-  return ["cpp", "java", "csharp"].includes(language);
+  return (
+    ["cpp", "java", "csharp"].includes(language) ||
+    (process.env["CODEQL_EXTRACTOR_GO_TRACE"] === "on" &&
```

Comment:
```suggestion
    (process.env["CODEQL_EXTRACTOR_GO_BUILD_TRACING"] === "on" &&
```

We decided to rename it in the other PR for clarity.